src/wildcard-service/init.lua

```diff
@@ -6,4 +11,67 @@ local mt = {
   __index = _M
 }
 
+function _M.new(options)
+  local opts = options or {}
+  local backend = cache_backend.new(opts.client or resty_backend)
```

Comment:
This will fail when `opts.client` is nil, right? `resty_backend` is undefined global variable.